modules/services/parcellite.nix

```diff
@@ -5,17 +5,23 @@ with lib;
 let
 
   cfg = config.services.parcellite;
-  package = pkgs.parcellite;
 
 in {
   meta.maintainers = [ maintainers.gleber ];
 
-  options = {
-    services.parcellite = { enable = mkEnableOption "Parcellite"; };
+  options.services.parcellite = {
+    enable = mkEnableOption "Parcellite";
+
+    package = mkOption {
+      description = "Parcellite derivation to use.";
+      type = types.package;
+      default = pkgs.parcellite;
+      defaultText = literalExample "pkgs.clipit";
```

Comment:
But `pkgs.clipit` is not the default? Perhaps you meant this as an example?

Reply:
You're absolutely right; fixed